Updating liquibase template

twerthi opened the pull request:
---

# Background

This update adds support for SQL Anywhere (Sybase) deployments.  In addition, it makes the downloading of the database drivers optional and not dependent on the download of Liquibase itself.  This allows the `octopuslabs/liquibase-workertools` to only download the drivers for execution.

# Results

Deployments against SQL Anywhere (Sybase) are now supported.  Warning, this could potentially be a breaking change.

## Before

Database deployments to SQL Anywhere were not supported.

## After

Now they are!

# Pre-requisites

- [X] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [X] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [X] Parameter names should not start with `$`
- [X] **Step template parameter names (the ones declared in the JSON, not the script body) should be prefixed with a namespace so that they are less likely to clash with other user-defined variables in Octopus** (see [this issue](https://github.com/OctopusDeploy/Issues/issues/2126)). For example, use an abbreviated name of the step template or the category of the step template).
- [X] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [X] The best practices documented [here](https://github.com/OctopusDeploy/Library/wiki/Best-Practices) have been applied
- [ ] If a new `Category` has been created:
   - [ ] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [ ] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

twerthi commented:
I forgot to mention that Liquibase will be removing packaged driver files for the Community version.  Downloading the drivers was tied to downloading Liquibase itself.  The functionality that was moved was put in place for when the removal of the drivers occurs so users could simply tick the box and it would work as it did before.

benPearce1 commented:
@twerthi this is a bit of a confusing change. 

The `liquibaseDownloadDatabaseDriver` value is provided by a docker container that the script is running inside of, correct?

Is there a reason that the block at line 975 was commented out?

twerthi commented:
Latest version of Liquibase requires Java 17 or higher, updated the version of Java downloaded when choosing the Download Liquibase option.  Updated default value for Download Database Driver to true.
